src/main/java/uk/gov/ons/census/casesvc/model/entity/Case.java

```diff
@@ -76,6 +77,9 @@
 
   @Column private String actionPlanId;
 
+  @Column(columnDefinition = "timestamp with time zone")
+  private Date createdDateTime;
```

Comment:
This should be `OffsetDateTime`

Reply:
also don't need the `(columnDefinition = "timestamp with time zone")` part

Reply:
how will postgres hold time zone without columnDefinition?

Reply:
OffsetDateTime implies time zone

Reply:
done